libsolidity/ast/Types.cpp

```diff
@@ -147,6 +147,21 @@ bool fitsIntoBits(bigint const& _value, unsigned _bits, bool _signed)
 	));
 }
 
+TypePointers transformParametersToExternal(TypePointers const& _parameters, bool _inLibrary)
+{
+	TypePointers transformed;
+
+	for (auto type: _parameters)
+	{
+		if (auto ext = type->interfaceType(_inLibrary).get())
```

Comment:
```suggestion
		if (TypePointer ext = type->interfaceType(_inLibrary).get())
```

otherwise, the `.get()` looks like the `.get` function on shared_ptr and thus it might seem like `ext` is a raw pointer.